browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -148,63 +151,71 @@ test.describe('applicant program index page', () => {
 
-    await validateScreenshot(page, 'common-intake-form-sections')
-    await applicantQuestions.expectPrograms({
-      wantNotStartedPrograms: [otherProgramName],
-      wantInProgressPrograms: [primaryProgramName],
-      wantSubmittedPrograms: [],
+      await loginAsAdmin(page)
+      await adminPrograms.addProgram(
+        commonIntakeFormProgramName,
+        'program description',
+        'https://usa.gov',
+        ProgramVisibility.PUBLIC,
+        'admin description',
+        /* isCommonIntake= */ true,
+      )
+      await adminPrograms.publishAllDrafts()
+      await logout(page)
     })
-    await applicantQuestions.expectCommonIntakeForm(commonIntakeFormProgramName)
-    await validateAccessibility(page)
-  })
 
-  test('shows a different title for the common intake form', async () => {
-    const {page, applicantQuestions} = ctx
-    await enableFeatureFlag(page, 'intake_form_enabled')
+    test('shows common intake form', async ({page, applicantQuestions}) => {
+      await applicantQuestions.applyProgram(primaryProgramName)
+      await applicantQuestions.answerTextQuestion('first answer')
+      await applicantQuestions.clickNext()
+      await applicantQuestions.gotoApplicantHomePage()
+
+      await validateScreenshot(page, 'common-intake-form-sections')
+      await applicantQuestions.expectPrograms({
+        wantNotStartedPrograms: [otherProgramName],
+        wantInProgressPrograms: [primaryProgramName],
+        wantSubmittedPrograms: [],
+      })
+      await applicantQuestions.expectCommonIntakeForm(
+        commonIntakeFormProgramName,
+      )
+      await validateAccessibility(page)
+    })
 
-    await applicantQuestions.clickApplyProgramButton(primaryProgramName)
-    expect(await page.innerText('h2')).toContain('Program application summary')
+    test('shows a different title for the common intake form', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.clickApplyProgramButton(primaryProgramName)
+      expect(await page.innerText('h2')).toContain(
+        'Program application summary',
+      )
 
-    await applicantQuestions.gotoApplicantHomePage()
-    await applicantQuestions.clickApplyProgramButton('Benefits finder')
-    expect(await page.innerText('h2')).toContain(
-      'Benefits pre-screener summary',
-    )
+      await applicantQuestions.gotoApplicantHomePage()
+      await applicantQuestions.clickApplyProgramButton('Benefits finder')
+      expect(await page.innerText('h2')).toContain(
+        'Benefits pre-screener summary',
+      )
+    })
   })
 
-  test('shows previously answered on text for questions that had been answered', async () => {
-    const {page, applicantQuestions} = ctx
-
+  test('shows previously answered on text for questions that had been answered', async ({
+    page,
+    applicantQuestions,
+  }) => {
     // Fill out application with one question and confirm it shows previously answered at the end.
```

Comment:
**suggestion (non-blocking):** We could replace the comments with test.step here and below.

Reply:
Decided against it to ease migration (we generally haven't been doing that).